frontend/libs/data-domain/store/src/lib/clients/team-data/team-data.client.ts

```diff
@@ -14,6 +14,16 @@ export class TeamDataClient {
   constructor(private http: HttpClient) {}
 
   public getTeamData$(teamId: number): Observable<TeamData> {
-    return this.http.get<TeamData>(ENDPOINT + teamId.toString());
+    return this.http.get<TeamData>(ENDPOINT + teamId.toString()).pipe(
+      map((response) => ({
+        ...response,
+        createdAt: new Date(response.createdAt),
```

Comment:
Ggf. können wir das BE-Seitig auch ändern und dann müssten wir das hier nicht machen